Crane/buildSrc/src/main/java/com/example/crane/buildsrc/Dependencies.kt

```diff
@@ -59,7 +59,7 @@ object Libs {
         object Compose {
             const val compilerVersion = "1.2.0"
```

Comment:
Should we also consider upgrading the compiler version to 1.3.0-rc02? That includes a Kotlin upgrade from 1.7.0 to 1.7.10. Might be nice to have that latest version in here as well.

Reply:
Good point, done!